lib/rataextra-backend.ts

```diff
@@ -62,6 +64,54 @@ export class RataExtraBackendStack extends NestedStack {
       securityGroups: securityGroups,
     };
 
+    const prismaParameters = {
+      ...genericLambdaParameters,
+      environment: {
+        SSM_DATABASE_NAME_ID: SSM_DATABASE_NAME,
+        SSM_DATABASE_DOMAIN_ID: SSM_DATABASE_DOMAIN,
+        SSM_DATABASE_PASSWORD_ID: SSM_DATABASE_PASSWORD,
+        DATABASE_URL: '',
+      },
+      bundling: {
+        nodeModules: ['prisma', '@prisma/client'],
+        commandHooks: {
+          beforeInstall(inputDir: string, outputDir: string) {
+            return [`cp -R ${inputDir}/packages/server/prisma ${outputDir}/`];
+          },
+          beforeBundling(_inputDir: string, _outputDir: string) {
+            return [];
+          },
+          afterBundling(_inputDir: string, outputDir: string) {
+            return [
+              `cd ${outputDir}`,
+              'npx prisma generate',
+              'rm -rf node_modules/@prisma/engines',
+              'rm -rf node_modules/@prisma/client/node_modules node_modules/.bin node_modules/prisma',
+            ];
+          },
+        },
+      },
+    };
+
+    const ssmParameterPolicy = new PolicyStatement({
+      actions: ['ssm:GetParameter', 'ssm:GetParameters'],
+      resources: [
+        `arn:aws:ssm:eu-west-1:178238255639:parameter/${SSM_DATABASE_DOMAIN}`,
+        `arn:aws:ssm:eu-west-1:178238255639:parameter/${SSM_DATABASE_NAME}`,
+        `arn:aws:ssm:eu-west-1:178238255639:parameter/${SSM_DATABASE_PASSWORD}`,
+      ],
+    });
+
+    const ssmDescribeParametersPolicy = new PolicyStatement({
+      actions: ['ssm:DescribeParameters'],
+      resources: ['*'],
+    });
+
+    const ksmDecryptPolicy = new PolicyStatement({
+      actions: ['kms:Decrypt'],
+      resources: ['arn:aws:kms:eu-west-1:178238255639:key/6cd436ad-f1f8-479f-aa56-da5a3f7a0711'],
```

Comment:
Later, this might be something to be defined higher in the stack and then passed down, if other resources need access to this.
👍 for specifying instead of using *!

Reply:
"These values" include the key being decrypted, as mentioned in the first comment.

Reply:
Instead of using a Policy, you could import the key with alias (something like `const ssmDecryptKeyAlias = kms.Alias.fromAliasName(this, 'myKey', 'alias/aws/ssm')`) and then use `ssmDecryptKeyAlias.grantDecrypt(lambda)` to grant decryption rights.